packages/components/src/search-control/style.scss

```diff
@@ -18,7 +18,8 @@
 
 		&:focus {
 			background: $white;
-			box-shadow: inset 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
+			box-shadow: inset 0 0 0 var(--wp-admin-border-width-focus) $components-color-accent;
+			border-color: $components-color-accent;
```

Comment:
`border-color` was previously defined by `@wordpress/base-styles` for`:focus`. For now, an override has been added to the component styles to allow theming to take effect.